src/templates/app/cashier/index.jsx

```diff
@@ -116,9 +116,9 @@ const Cashier = () => (
                     <a className='invisible normal_currency client_logged_out' href={it.url_for('cashier/payment_methods')} id='view_payment_methods'>
                         <span>{it.L('View available payment methods')}</span>
                     </a>
-                    <CashierNote className='gr-hide-m gr-child' text={it.L('Please do not share your bank account, credit card, or e-wallet with another client, as this may cause delays in your withdrawals')} />
+                    <CashierNote className='gr-hide-m gr-child' text={it.L('Please do not share your bank account, credit card, or <br> e-wallet with another client, as this may cause delays in your withdrawals')} />
                 </div>
-                <CashierNote className='gr-12 gr-hide gr-show-m gr-child' text={it.L('Please do not share your bank account, credit card, or e-wallet with another client, as this may cause delays in your withdrawals')} />
+                <CashierNote className='gr-12 gr-hide gr-show-m gr-child' text={it.L('Please do not share your bank account, credit card, or <br> e-wallet with another client, as this may cause delays in your withdrawals')} />
```

Comment:
please handle it differently, this will send `<br>` tag to translators

Reply:
I removed the <br> tag. Instead uses &#x2011;